### PR TITLE
fix(core): fix `alter table resume wal` not having the effect

### PR DESCRIPTION
**Pull request by ideoma**
Sometimes after running 

```
ALTER TABLE x RESUME WAL
```

the table is left in unsuspended state but the transactions are still not applied until the restart

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 10 / 10 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/wal/OperationExecutor.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/ApplyWal2TableJob.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 10 / 10 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/wal/OperationExecutor.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/ApplyWal2TableJob.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 10 / 10 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/wal/OperationExecutor.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/ApplyWal2TableJob.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|2|2|100.00%|

File: core/src/main/java/io/questdb/cairo/wal/seq/TableSequencerAPI.java [hunk 1]
```diff
@@ -357,6 +357,9 @@ public void resumeTable(TableToken tableToken, long resumeFromTxn) {
         try (TableSequencerImpl sequencer = openSequencerLocked(tableToken, SequencerLockType.WRITE)) {
             try {
                 if (!sequencer.isSuspended()) {
+                    // Even if the table already unsuspended, send ApplyWal2TableJob notification anyway
+                    // as a way to resume table which is not moving even if it's marked as not suspended.
+                    sequencer.resumeTable();
```
Comment (marregui) [inline core/src/main/java/io/questdb/cairo/wal/seq/TableSequencerAPI.java:362]: like a notifyAll in thread synchronization?

Comment (ideoma) [reply core/src/main/java/io/questdb/cairo/wal/seq/TableSequencerAPI.java:362]: It doesn't hurt to notify Apply job about the table when the user issues `RESUME` SQL so there is no reason to optimise the job notification in this case.

**Review by marregui — APPROVED** [review]
LGTM

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 10 / 10 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/wal/OperationExecutor.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/ApplyWal2TableJob.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|2|2|100.00%|

**Pull request merged**